portfolio/src/main/webapp/style.css

```diff
@@ -1,23 +1,48 @@
+/* Define color scheme of page */
 :root {
     --primary: #ffcece;
     --primary-dark: #ffc1c8;
     --accent: #ffe3b0;
     --contrast: #8ed6ff;  
 }
-
+/* BACKGROUND IMAGES STYLES */
+#body {
+    background-position: center;
+    background-image: url(/images/background/sunset3.jpg);
```

Comment:
Seems like you only need sunset3.jpg -- why do you upload sunset and sunset2?

Reply:
Oh haha I was indecisive and didn't know which one I wanted. I'll remove the other two though. Also I got it from 
https://wallpapersafari.com/w/IhfHrj -- do you think this is okay?

Reply:
It is hard to say since I am not a lawyer :D 

Based on my reading of https://wallpapersafari.com/page/copyright-policy/, the content belongs to the author; and when you click "Image info" it shows "Crediting isn’t required, but is appreciated and allows us to gain exposure." It does not explicitly talk about licensing though, so I cannot give you an answer.

I think asking internteam for advise might be a good idea anyway.

Reply:
Okay! I just removed that image and added one I took (it looks less good but it's fine haha)